core/block/editor/clipboard/paste.go

```diff
@@ -292,7 +296,12 @@ func (p *pasteCtrl) insertUnderSelection() (err error) {
 
 func (p *pasteCtrl) removeSelection() {
 	for _, toRemove := range p.selIds {
-		p.s.Unlink(toRemove)
+		if toRemove != template.TitleBlockId &&
```

Comment:
May be extract this condition to seperated function to make it clearer?

Reply:
agreed